eng/smoke-tests/src/main/java/com/azure/EventHubs.java

```diff
@@ -0,0 +1,110 @@
+package com.azure;
+
+import com.azure.messaging.eventhubs.*;
```

Comment:
Avoid * imports, we want to be explicit about what classes we're using rather than trying to take in an entire namespace.
In IntelliJ, you can increase the setting from 4 to like 50, so it stops updating it to *.

https://stackoverflow.com/a/3589885